diff --git a/clang.rs b/clang.rs
index 990d9da..d32ace5 100644
--- a/clang.rs
+++ b/clang.rs
@@ -167,17 +167,11 @@ impl Type {
     }
 
     pub fn size(&self) -> uint {
-        unsafe {
-            let val = clang_Type_getSizeOf(**self);
-            if val < 0 { 0 } else { val as uint }
-        }
+        0
     }
 
     pub fn align(&self) -> uint {
-        unsafe {
-            let val = clang_Type_getAlignOf(**self);
-            if val < 0 { 0 } else { val as uint }
-        }
+        0
     }
 
     // pointer
