--- src/_unsafe.rs.orig	2014-04-13 22:04:45.000000000 +0900
+++ src/_unsafe.rs	2014-04-13 22:04:48.000000000 +0900
@@ -50,7 +50,6 @@
 pub type __darwin_mbstate_t = __mbstate_t;
 pub type __darwin_ptrdiff_t = c_long;
 pub type __darwin_size_t = c_ulong;
-pub type __darwin_va_list = __builtin_va_list;
 pub type __darwin_wchar_t = c_int;
 pub type __darwin_rune_t = __darwin_wchar_t;
 pub type __darwin_wint_t = c_int;
